changelog/README.md

```diff
@@ -4,6 +4,7 @@
 
 ### 4.4.0 (in progress)
 
+- [improvement] JAVA-2561: Mark DefaultUdtValue and DefaultTupleValue as not thread safe
```

Comment:
Since the changes are internal I'm not sure if it's worth mentioning in the changelog.